.github/workflows/ci.yaml

```diff
@@ -48,3 +48,27 @@ jobs:
       - name: Run tests
         run: yarn test
 
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      FOUNDRY_PROFILE: ci
+      ETH_RPC_URL: ${{ secrets.ETH_RPC_URL }}
+      INFURA_ID: ${{ secrets.INFURA_ID }}
+      ETHERSCAN_API_KEY: ${{ secrets.ETHERSCAN_API_KEY }}
+      POLYGONSCAN_API_KEY: ${{ secrets.POLYGONSCAN_API_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - uses: volta-cli/action@v1
+
+      - name: Install Foundry
+        uses: onbjerg/foundry-toolchain@v1
+        with:
+          version: nightly
+
+      - run: yarn install --network-concurrency 1
```

Comment:
Why do we need `--network-concurrency 1`?

Reply:
I will remove